src/components/templates/Audience/Audience.tsx

```diff
@@ -369,25 +369,28 @@ export const Audience: React.FC<AudienceProps> = ({ venue }) => {
             Leave Seat
           </button>
         </div>
-        <div className="shout-container">
-          <form onSubmit={handleSubmit(onSubmit)} className="shout-form">
-            <input
-              name="text"
-              className="text"
-              placeholder="Shout out to the crowd"
-              ref={register({ required: true })}
-              disabled={isShoutSent}
-              autoComplete="off"
-            />
-            <input
-              className={`shout-button ${isShoutSent ? "btn-success" : ""} `}
-              type="submit"
-              id={`send-shout-out-${venue.name}`}
-              value={isShoutSent ? "Sent!" : "Send"}
-              disabled={isShoutSent}
-            />
-          </form>
-        </div>
+
+        {venue.showShoutouts && (
+          <div className="shout-container">
```

Comment:
See further discussion about this on the referenced comment:

> I believe shout-outs are intended to be part of more templates than just the audience? At least they used to be (even though I believe they were hackily commented out from a bunch of them) So we may just want to do this in all cases rather than for this specific venue type?
> 
> _Originally posted by @0xdevalias in https://github.com/sparkletown/sparkle/pull/1687#r660310188_